changelog/7496.improvement.md

```diff
@@ -0,0 +1,36 @@
+Make TED an end-to-end policy. Namely, make it possible to train TED on stories that contain
```

Comment:
let's also link the official docs page for it in the changelog.

Reply:
how to add doc link in changelog? I didn't find it in the description

Reply:
Should be something like `(my link test)[policies.mdx#some-section]`

Reply:
will it know address of `policies.mdx` form where changelog is?

Reply:
Yes 👍🏻  If not -> CI fails